--- a/debian/control	2022-07-27 13:52:07.000000000 +0000
+++ b/debian/control	2024-05-22 21:24:58.492095011 +0000
@@ -33,7 +33,7 @@
 Vcs-Git: https://salsa.debian.org/cloud-team/ignition.git
 Homepage: https://github.com/coreos/ignition
 Rules-Requires-Root: no
-XS-Go-Import-Path: github.com/coreos/ignition
+XS-Go-Import-Path: github.com/flatcar/ignition/v2

 Package: ignition
 Architecture: any
diff -Naur debian/rules debian2/rules
--- a/debian/rules	2022-07-27 13:52:07.000000000 +0000
+++ b/debian/rules	2024-05-22 21:27:21.278367008 +0000
@@ -3,8 +3,9 @@
 include /usr/share/dpkg/default.mk

 export DH_GOLANG_BUILDPKG := \
-    github.com/coreos/ignition/internal \
-    github.com/coreos/ignition/validate
+    github.com/flatcar/ignition/v2/internal \
+    github.com/flatcar/ignition/v2/validate
+export GO15VENDOREXPERIMENT="1"

 %:
 	dh $@ --builddirectory=_build --buildsystem=golang --with=golang
@@ -14,6 +15,6 @@

 override_dh_auto_build:
 	dh_auto_build -- \
-		-ldflags "-X 'github.com/coreos/ignition/internal/version.Raw=$(DEB_VENDOR) $(DEB_VERSION)'"
+		-ldflags "-X 'github.com/flatcar/ignition/v2/internal/version.Raw=$(DEB_VENDOR) $(DEB_VERSION)'"

 override_dh_auto_test::